src/net_processing.cpp

```diff
@@ -2390,8 +2397,12 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
         }
         bool fNewBlock = false;
         ProcessNewBlock(chainparams, pblock, forceProcessing, &fNewBlock);
-        if (fNewBlock)
+        if (fNewBlock) {
```

Comment:
I believe that @TheBlueMatt had plans to clean this up further (or remove the need) in the future, so if it's all the same, I'd prefer to leave it as-is for 0.15.